packages/ui/src/helpers/authenticator/__tests__/listenToAuthHub.test.ts

```diff
@@ -75,6 +77,30 @@ describe('listenToAuthHub', () => {
     Hub.dispatch('auth', { event: 'tokenRefresh_failure' });
     expect(sendSpy).not.toHaveBeenCalled();
 
+    unsubscribe();
+  });
+  it('ignores autoSignIn event when state is authenticated', () => {
```

Comment:
Very nit, let's put a new line between `it`s. Same below!

Reply:
Done